docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst

```diff
@@ -2,7 +2,7 @@
 
 How to cross compile using Qt Creator and PELUX SDE
 ===================================================
-.. taglist:: howto
+.. tags:: howto
```

Comment:
It should be `.. tag:: howto`

Reply:
But if you look at other articles, there is "tags" used, since you can use comma to separate multiple tags, right?

Reply:
The tag directive is used to tag articles. The taglist directive is used to list articles matching a certain tag.

Reply:
My bad! It was the documentation which was wrong.

Reply:
Great we found a bug in the doc! :)